### ISSUE TITLE
Need to get rid of database declarations in SQL file

### ISSUE THREAD

**Issue by Mcghee-Jake** [team-makeSmart/CST336#19]
So when using cloud 9, you are allowed to create and use multiple databases without any issues. Such as
![capture](https://user-images.githubusercontent.com/33072387/43490496-bbd5edcc-94d5-11e8-8c45-5d3f2b469ccd.JPG)

However, when migrating to Heroku, we will only be allowed to use the default heroku database (Its called something like heroku_[somerandomtexthere].

![capture2](https://user-images.githubusercontent.com/33072387/43490529-e8e5e362-94d5-11e8-86ad-978e5dc6f19e.JPG)

So when we execute something like 
![capture3](https://user-images.githubusercontent.com/33072387/43490553-fe825660-94d5-11e8-8233-d3f09e8f72eb.JPG)
We wont be able to access that database from within our heroku webpage since inside of heroku all the tables need to be stored inside of the heroku_[somerandomtexthere] database.

Originally I thought we could maybe avoid this just by ignoring this stuff
![capture4](https://user-images.githubusercontent.com/33072387/43490667-6d6ef63c-94d6-11e8-9544-538a01176e38.JPG)
and copying/pasting everything under it when we migrate to heroku. But since the create table statements also contain the database name, such as 
![capture5](https://user-images.githubusercontent.com/33072387/43490713-937e9d78-94d6-11e8-8005-c2dec21c1731.JPG)

Basically we need to keep all the tables the same but make it so that the database name is ambiguous.  It might be easier to make this change manually rather than trying to do it in the SQL program, but i'll leave that up to you.

**Comment by mdoussias** [comment]
This task is completed.  Thank you Jake for catching that bug.  Would have been annoying last minute scrambling.
